packages/icons/stories/icon.doc.stories.mdx

```diff
@@ -55,7 +55,7 @@ To override the default color of the icon, you can use the `color` css variable.
   />
 </Canvas>
 
-You can supply custom icons as `children` of the `Icon` component by using an `<svg>`.
+You can supply custom icons as 'children' of the `Icon` component by using an `<svg>`.
```

Comment:
Just curious about this change. `children` is a special value when dealing with components.

Also @Fercas123 seems like this still needs to be updated

Reply:
my mistake